CMakeLists.txt

```diff
@@ -2,7 +2,13 @@ cmake_minimum_required(VERSION "3.8")
 
 project("Modern C++ Kafka API" VERSION 1.0.0)
 
-include(CTest)
+get_property(parent_directory DIRECTORY PROPERTY PARENT_DIRECTORY)
+if(NOT parent_directory)
+    set(cppkafka_master_project ON)
```

Comment:
How about renaming `cppkafka_master_project` to `TO_BUILD_ADDTIONAL_BINARIES`? Only one flag, for all these tests/tools/examples. Then we don't need `CPPKAFKA_ENABLE_TESTS` either.

Reply:
The naming is kinda standardized, when looking for popular modern-cmake projects. For example: `https://github.com/fmtlib/fmt/blob/master/CMakeLists.txt`. Also lowercase is specifically for non-cache variables and UPPER_CASE with `PROJECT_NAME_` prefix is often used for cache variables (which implicitly options are).  But this is convention, you don't have to follow it. The main purpose for this PR is to make it easy to use with ExternalProject, not to make the CMake pretty :)

Reply:
Now I see. Thank you! 😀